src/lib/components/token-field/stories/index.stories.tsx

```diff
@@ -0,0 +1,204 @@
+import { Meta, StoryObj } from '@storybook/react';
+
+import { TokenSVG } from '../../../icons';
+import { TokenField } from '..';
+
+const meta: Meta<typeof TokenField> = {
+  title: 'TokenField',
+  component: TokenField,
+  argTypes: {
+    disabled: {
+      defaultValue: false,
+      control: { type: 'boolean' },
+    },
+    labelPosition: {
+      defaultValue: 'right',
+      control: { type: 'radio' },
+      options: ['left', 'right'],
+    },
+    textAlign: {
+      defaultValue: 'right',
+      control: { type: 'radio' },
+      options: ['left', 'right'],
+    },
+  },
+};
+
+export default meta;
+
+type Story = StoryObj<typeof TokenField>;
+
+export const FilledWithToken: Story = {
```

Comment:
All text fields should be the same size but text field default, with top label and text field disabled, are of different sizes.